### PR TITLE
Move marketing year to invoice line

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: app/processing/get-completed-payment-requests-filter.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ const getCompletedPaymentRequestsFilter = (paymentRequest) => {
       return {
         schemeId: paymentRequest.schemeId,
         frn: paymentRequest.frn,
-        marketingYear: paymentRequest.marketingYear,
```
Comment (Kirsten1822) [inline app/processing/get-completed-payment-requests-filter.js:16]: Why was this removed here?

Comment (johnwatson484) [reply app/processing/get-completed-payment-requests-filter.js:16]: The reason for the ticket is that a CS payment request can have multiple marketing years in it's invoice lines.  One of them will be shown in the payment request level property sure, but if we restrict the filter to only those with the same year, we may risk missing a previous payment request because the year on the payment request happened to be different.

Fortunately, the way CS works, we can use the contract number and FRN to still get all previous payment requests.

Comment (Kirsten1822) [reply app/processing/get-completed-payment-requests-filter.js:16]: Ah okay, good :)
